src/components/HorizontalBarChart/index.js

```diff
@@ -0,0 +1,31 @@
+import './styles.css';
+
+const HorizontalBarChart = ({ data, maxValue, title }) => (
+  <div className="bar-chart-container ">
+    <h2 className="bar-chart-title">{title}</h2>
+    {data.map((value, index) => {
+      let numericValue = value;
+      let displayValue;
+      if (typeof value !== 'number') {
+        numericValue = value[1];
+        displayValue = value[0];
+      }
+      const widthPercentage = (numericValue * 100) / maxValue ?? 10;
+
+      return (
+        <div className="bar-container" key={index}>
```

Comment:
Don't you have an identifier to use as `key`?

Reply:
Only for the array case (it is the word string), for the guess distribution the index is the key because is the number of guesses minus 1.